browser/brave_browser_features.cc

```diff
@@ -12,4 +12,9 @@ BASE_FEATURE(kBraveCleanupSessionCookiesOnSessionRestore,
              "BraveCleanupSessionCookiesOnSessionRestore",
              base::FEATURE_ENABLED_BY_DEFAULT);
 
+// Sanitize url before copying, replaces default ctrl+c hotkey for urls.
+BASE_FEATURE(kBraveCopyCleanLink,
```

Comment:
To go with my other suggestion: `kBraveCopyCleanLinkByDefault`

Reply:
done